pyanaconda/ui/tui/spokes/user.py

```diff
@@ -182,25 +199,24 @@ def completed(self):
     @property
     def showable(self):
         return not (self.completed and flags.automatedInstall
-                    and self.data.user.seen and not self._policy.changesok)
+                    and self._user_requested and not self._policy.changesok)
 
     @property
     def mandatory(self):
-        """ Only mandatory if the root pw hasn't been set in the UI
-            eg. not mandatory if the root account was locked in a kickstart
-        """
-        return not self._users_module.proxy.IsRootPasswordSet and not self._users_module.proxy.IsRootAccountLocked
+        """Only mandatory if no admin user has been requested."""
+        return not self._users_module.AdminUserExists
 
     @property
     def status(self):
-        if len(self.data.user.userList) == 0:
+        user_list = get_user_list(self._users_module)
+        if len(user_list) == 0:
             return _("No user will be created")
-        elif self._use_password and not bool(self._user_data.password or self._user_data.isCrypted):
+        elif self._use_password and not bool(self._user_data.password or self._user_data.is_crypted):
             return _("You must set a password")
-        elif "wheel" in self.data.user.userList[0].groups:
-            return _("Administrator %s will be created") % self.data.user.userList[0].name
+        elif self._user_data.has_admin_priviledges():
```

Comment:
You are mixing `user_list` with `self._user_data` in this property.

Reply:
Good point! Should be fixed now.